.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "CommunityToolkit"]
+	path = CommunityToolkit
```

Comment:
Maybe call this `CommunityToolkitDocs` or just `docs_submodule` or something? Not sure if there's a good convention, but would be good to clarify that there's docs in here in the path, though I know the interesting subfolder is also just called `docs`.

Reply:
Renamed to "msdocs-communitytoolkit". Used lowerCamelCase for consistency with other folder names, I saw ImageSharp is doing the same for their "shared-infrastructure" submodule as well. Plus this name is clearer than before, thanks! 🙂